### PR TITLE
Feature challenge01

### PR DESCRIPTION
**Pull request by jagyaseniDas**
In katacode, install and configure of apache2 web server done.

### CODE REVIEW

**Review by zeusbaba — APPROVED** [review]

File: challenge-01/1 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ansible in katacode
```
Comment (zeusbaba) [inline challenge-01/1]: plz remove this file, because it doesnt belong

File: challenge-01/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+ansible in katacode
```
Comment (zeusbaba) [inline challenge-01/README.md]: plz use markdown to make it simpler and readable 

see an example  
```
this is a code snippet  
print etc etc
```

use a headline
## this is a headline

File: challenge-01/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+Testing code First time
+
+Install and configure Apache2 web server
```
Comment (zeusbaba) [inline challenge-01/README.md]: codesnippet on how to run this

**Review by zeusbaba — CHANGES_REQUESTED** [review]

File: challenge-01/1 (hunk 1 above)
Comment (jagyaseniDas) [reply challenge-01/1]: Removed the above said file.

**Review by zeusbaba — APPROVED** [review]
well done

**Comment by zeusbaba** [comment]
it looks good, feel free to merge it.  
tip: `squash & merge` is preferred

**Pull request merged**